packages/calypso-build/webpack/test/util.js

```diff
@@ -0,0 +1,23 @@
+const { cssNameFromFilename } = require( '../util' );
+
+describe( 'cssNameFromFilename()', () => {
+	test( 'Transforms static string', () => {
+		expect( cssNameFromFilename( 'static.js' ) ).toBe( 'static.css' );
+	} );
+
+	test( 'Passes `undefined` through', () => {
+		expect( cssNameFromFilename() ).toBeUndefined();
+	} );
+
+	test( 'Handles templates', () => {
+		expect( cssNameFromFilename( '[name].[hash].js' ) ).toBe( '[name].[hash].css' );
+	} );
+
+	test( 'Maintains query string', () => {
+		expect( cssNameFromFilename( 'static.js?query=string' ) ).toBe( 'static.css?query=string' );
+	} );
+
+	test( 'Returns other extensions untouched 😬', () => {
+		expect( 'static.jsx' ).toBe( 'static.jsx' );
```

Comment:
I _think_ this is OK. I don't think you ever want to get an extension other than `js` out of webpack.